chainerrl/agent.py

```diff
@@ -107,7 +119,7 @@ def save(self, dirname):
     def load(self, dirname):
         """Load internal states."""
         for attr in self.saved_attributes:
-            serializers.load_npz(
+            load_npz_no_strict(
```

Comment:
What is the reason behind this change? It's better to add comments if this is a fix for some problem.